test/unit/lua_msgpack.c

```diff
@@ -123,6 +160,7 @@ main(void)
 	lua_pop(L, 1);
 
 	test_translation_in_encoding(L);
+	test_dict_in_indexation(L);
```

Comment:
Nit: I'd rename all other occurrences of `dict` to `translation` as well for consistency (including `obj->dict`).

Reply:
Sounds reasonabled, fixed.